acls.go

```diff
@@ -123,6 +124,144 @@ func (h *Headscale) UpdateACLRules() error {
 	return nil
 }
 
+func (h *Headscale) ListACLPolicy() (*ACLPolicy, error) {
+	return h.aclPolicy, nil
+}
+
+func ACLProtoToStruct(v *v1.ACLPolicy) (*ACLPolicy, error) {
```

Comment:
Here the linter is complaining

> parameter name 'v' is too short for the scope of its usage (varnamelen)


Replace `v`for `policy` or something like this, for instance.